layouts/page/cuvs.html

```diff
@@ -0,0 +1,153 @@
+        </div>
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-users-viewfinder"></i> Computer Vision </h2>
+          <p>For computer vision tasks like image recognition or object detection, vector search aids in quickly finding similar vectors, thus accelerating the classification process. With GPU-accelerated vector search, the process of matching features and identifying patterns is significantly optimized, enhancing computer vision systems' overall speed and accuracy. </p>
+        </div>
+        <div class="col-md-3 py-3">
+          <h2><i class="fa-light fa-cloud-binary"></i> Data Mining </h2>
+          <p>Vector search forms the backbone for many important data mining algorithms, such as finding clusters, reducing dimensionality, and visualizing data points. GPU-accelerated vector search can take large machine learning and natural language processing workflows from hours to near real-time speeds, enabling important use-cases like hybrid search and anomaly detection. </p>
+        </div>
+      </div>
+
+
+      <h1 class="mt-10 mb-4"> Learn More</h1>
+      <p class="subheading  mb-5">Find where to get more resources and connect with developers.</p>
+
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-flask-gear"></i> Open Beta</h2>
+          <p>cuVS is in open beta and under active development. It's ready for wide use, but has a few known limitations and edge cases. You can learn more through the <a href="https://docs.rapids.ai/api/raft/stable/" target="_blank"> documentation</a>, which also includes instructions for running <a href="https://docs.rapids.ai/api/raft/stable/raft_ann_benchmarks/" target="_blank"> cuVS ANN benchmarks</a> on your system.</p>
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-comment-code"></i> Get in Touch</h2>
+          </p> Prefer conference presentations to documentation, get all the details about how cuVS works in the <a href="https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.nvidia.com%2Fgtc%2Fsession-catalog%2F%3Ftab.allsessions%3D1700692987788001F1cG%26search%3Ds62339%23%2F&data=05%7C02%7Caenemark%40nvidia.com%7C25a97bbd4c2b4e40033e08dc46e153f6%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C638463180832438619%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=ZntMhBBhql3sb52DlAiH1%2FJaPALSbVx5F2pdT4Ncti8%3D&reserved=0" target="_blank">GTC session</a>. Want to contribute, join our <a href="https://rapids.ai/slack-invite" target="_blank"> RAPIDS slack channel</a> or share feedback on <a href="https://github.com/rapidsai/cuvs" target="_blank"> GitHub <i class="fa-solid fa-arrow-up-right"></i></a></p>
```

Comment:
Use the direct link rather than the Outlook "safe link" redirection. Also a couple of edits were missed from the previous round.
```suggestion
          </p> Prefer conference presentations to documentation, get all the details about how cuVS works in the <a href="https://www.nvidia.com/gtc/session-catalog/?tab.allsessions=1700692987788001F1cG&search=s62339" target="_blank">GTC session</a>. To contribute, join our <a href="https://rapids.ai/slack-invite" target="_blank"> RAPIDS Slack channel</a> or share feedback on <a href="https://github.com/rapidsai/cuvs" target="_blank"> GitHub <i class="fa-solid fa-arrow-up-right"></i></a></p>
```